src/main/java/com/study/stockmanagementstudycase/model/dto/response/WareHouseResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.study.stockmanagementstudycase.model.dto.response;
+
+import lombok.*;
+
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class WareHouseResponse {
+
```

Comment:
Bu kısımda geriye id değeri dönülmesi daha efektif olabilir. Bir front-end sayfasında hangi WareHouse üzerinde işlem yapıldığı bilinmek istenebilir. Ya da erişilen bir WareHouse için daha sonra bir güncelleme isteği atılacağı zaman id değeri belirtilerek atılacağı için yine id değerine ihtiyaç duyulacak.

Reply:
Burada ben ortak bir yapi dondurmek istedim her endpoint icin ResponseEntity<WareHouseResponse> gibi, cunku diger durumda ortak bir yapi olusmuyor.

Reply:
+